Framework/API/test/SpectrumDetectorMappingTest.h

```diff
@@ -116,8 +116,8 @@ class SpectrumDetectorMappingTest : public CxxTest::TestSuite {
   }
 
   void test_array_constructor_null_inputs() {
-    specnum_t specs[2];
-    detid_t detids[2];
+    specnum_t specs[2] = {0, 1};
+    detid_t detids[2] = {0, 1};
     TS_ASSERT_THROWS(SpectrumDetectorMapping(nullptr, detids, 10), const std::invalid_argument &);
```

Comment:
Does this still test for '`null_inputs`'?

Reply:
Yes the `nullptr` is the argument that it's checking to be null, and it uses either `specs` or `detids` for the other argument.

Reply:
right got it :+1: